### ISSUE TITLE
compile error

### ISSUE THREAD

**Issue by rjahn** [sibvisions/reactUI#520]
```
ERROR in src/main/components/editors/linked/UIEditorLinked.tsx:1128:49
TS2322: Type '"instant"' is not assignable to type 'ScrollBehavior | undefined'.
    1126 |                             el.classList.add('p-highlight');
    1127 |                             el.setAttribute('data-p-highlight', 'true');
  > 1128 |                             el.scrollIntoView({ behavior: "instant" });
         |                                                 ^^^^^^^^
    1129 |                         }
    1130 |                     }
    1131 |                 }}
```

**Comment by gimmixAT** [comment]
That is an issue with erronous typing in the typescript version we are using (4.x). This seems to be fixed in 5.1 upwards.
I'll add some casting / ts-ingore for the time being to not open that can of worms of updating the typescript version.

**Comment by gimmixAT** [comment]
error message should be gone now
